### PR TITLE
Reorder menu in service create modal

### PR DESCRIPTION
**Pull request by philipnrmn**
Before to the left; after to the right:

### CODE REVIEW

**Comment by philipnrmn** [comment]
retest this please

**Comment by philipnrmn** [comment]
run integration tests

**Comment by philipnrmn** [comment]
run integration tests 💃

**Comment by philipnrmn** [comment]
run integration tests

File: plugins/services/src/js/components/modals/NewCreateServiceModalForm.js [hunk 1]
```diff
@@ -328,19 +328,19 @@ class NewCreateServiceModalForm extends Component {
     if (this.state.isPod) {
       return [
         <TabButton id="networking" label="Networking" key="multinetworking" />,
-        <TabButton id="environment" label="Environment"
-          key="multienvironment" />,
+        <TabButton id="volumes" label="Volumes" key="multivolumes" />,
         <TabButton id="healthChecks" label="Health Checks"
           key="multihealthChecks" />,
-        <TabButton id="volumes" label="Volumes" key="multivolumes" />
+        <TabButton id="environment" label="Environment"
```
Comment (mlunoe) [inline plugins/services/src/js/components/modals/NewCreateServiceModalForm.js]: #nit This has conventionally been our style:
```js
<TabButton
  id="environment"
  label="Environment"
  key="multienvironment" />
```

**Comment by mlunoe** [comment]
Looks good, works great! One minor nit, but nothing that stops this PR

File: plugins/services/src/js/components/modals/NewCreateServiceModalForm.js (hunk 1 above)
Comment (jfurrow) [inline plugins/services/src/js/components/modals/NewCreateServiceModalForm.js]: I adjusted the newlines here (and the line above as well)

**Comment by jfurrow** [comment]
@philipnrmn I pushed a commit, hope you don't mind! :ship:

**Pull request merged**